drift_dev/lib/api/migrations.dart

```diff
@@ -214,3 +214,33 @@ class InitializedSchema {
   /// ```
   DatabaseConnection newConnection() => _createConnection();
 }
+
+/// Utility function used by generated tests to:
+/// 1. Create a database at a specific version
+/// 2. Insert data into the database
+/// 3. Migrate the database to a target version
+/// 4. Validate that the data is valid after the migration
+Future<void> testStepByStepigrations<OldDatabase extends GeneratedDatabase,
+        NewDatabase extends GeneratedDatabase>(
+    {required SchemaVerifier verifier,
```

Comment:
I wonder if this should be an instance method on `SchemaVerifier`?

Reply:
Just say the word...

Reply:
Let's make it an instance method :)

Reply:
Also I think it should be called `testWithDataIntegrity` or something since it's not really related to step-by-step migrations.

Reply:
Should I make this disable foreign keys and re-enable if they were enabled?